src/browser/bounds_changed_state_tracker.js

```diff
@@ -193,6 +212,10 @@ function BoundsChangedStateTracker(uuid, name, browserWindow) {
         return newClippedBounds;
     };
 
+    let checkTrackingApi = (groupLeader) => {
```

Comment:
Const and shorter:
`const checkTrackingApi = ({type}) => type === 'api' ? trackingAPI || true;`